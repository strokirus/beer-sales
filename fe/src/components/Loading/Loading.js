import React from 'react';
import {
  Wrapper,
} from '../../cssGlobals/globals';

/**
 * Component to Loading
*/
const Loading = () => (
  <Wrapper>
    <h1>Loading</h1>
  </Wrapper>
);

Loading.propTypes = {
};

Loading.defaultProps = {
};

export default Loading;
