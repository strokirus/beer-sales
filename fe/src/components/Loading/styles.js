import styled from 'styled-components';

export const Input = styled.input`
    width: 80%;
    outline: none;
    height: 15px;
    margin-top: 13px;
    padding: 8px 5px;
    box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.3);
    font-size: 16px;
    border: 1px solid rgba(0, 0, 0, 0.3);
`;

export default {
  Input,
};
