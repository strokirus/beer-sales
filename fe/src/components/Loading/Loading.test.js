import React from 'react';
import renderer from 'react-test-renderer';
import Loading from './Loading';

describe('(Component) Loading', () => {
  it('should render an Loading', () => {
    const tree = renderer.create(<Loading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
