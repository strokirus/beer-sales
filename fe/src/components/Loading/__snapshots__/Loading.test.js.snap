// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) Loading should render an Loading 1`] = `
<section
  className="sc-AxhCb jmvNtO"
>
  <h1>
    Loading
  </h1>
</section>
`;
